lineman/app/app.coffee

```diff
@@ -15,6 +15,9 @@ angular.module('loomioApp', ['ngNewRouter',
   renderer = new marked.Renderer()
   renderer.link = (href, title, text) ->
     "<a href='" + href + "' title='" + (title || text) + "' target='_blank'>" + text + "</a>";
+  renderer.paragraph = (text) ->
+    text = text.replace(/([\s(\[]+|^)@([a-zA-Z0-9]+)([\s)\]\,\?]|$)/g, "$1<a class='lmo-user-mention' href='/u/$2'>@$2</a>$3")
```

Comment:
Hrm... the twitter-text libraries validate usernames as twitter usernames, which would be slightly different from how we do usernames currently. For example, I think there's a character limit (14?) on twitter usernames. Maybe we just conform our usernames to twitter, but I'm a little wary of that.